Makefile

```
@@ -70,18 +75,16 @@ define GOLDEN_TEST_FAILURE_MSG
 endef
 export GOLDEN_TEST_FAILURE_MSG
 
-.PHONY: _chartsnap.kong
-_chartsnap.kong:
-	@ $(MAKE) _chartsnap GOLDEN_TEST_CHART=kong GOLDEN_TEST_CHART_VALUES_DIR=./charts/kong/ci/ \
-	CHARTSNAP_ARGS=$(CHARTSNAP_ARGS)
-
-.PHONY: _chartsnap.ingress
-_chartsnap.ingress:
-	@ $(MAKE) _chartsnap GOLDEN_TEST_CHART=ingress GOLDEN_TEST_CHART_VALUES_DIR=./charts/ingress/ci/ \
-	CHARTSNAP_ARGS=$(CHARTSNAP_ARGS)
-
 .PHONY: _chartsnap
-_chartsnap: chartsnap
-	@ helm repo update kong
-	@ helm dependencies update charts/ingress
-	@ helm chartsnap -c ./charts/$(GOLDEN_TEST_CHART) -f $(GOLDEN_TEST_CHART_VALUES_DIR) $(CHARTSNAP_ARGS)
+_chartsnap:
+	$(MAKE) _chartsnap.deps \
+		GOLDEN_TEST_CHART=$(CHART) \
+		GOLDEN_TEST_CHART_VALUES_DIR=./charts/$(CHART)/ci/ \
+		CHARTSNAP_ARGS=$(CHARTSNAP_ARGS)
+
+.PHONY: _chartsnap.deps
+_chartsnap.deps: chartsnap
+ifeq ($(CHART),"kong")
+	$(CHART) helm dependencies update charts/ingress
+endif
+	helm chartsnap -c ./charts/$(GOLDEN_TEST_CHART) -f $(GOLDEN_TEST_CHART_VALUES_DIR) $(CHARTSNAP_ARGS)
```

I've noticed `helm dependencies update charts/ingress` is not run when running `_chartsnap` target for `kong` - I think it's because `ifeq` in Makefile is not evaluated in runtime. Seems like this fixes it:
```suggestion
.PHONY: _chartsnap
_chartsnap: _chartsnap.deps
	helm chartsnap -c ./charts/$(CHART) -f ./charts/$(CHART)/ci/ $(CHARTSNAP_ARGS)

.PHONY: _chartsnap.deps
_chartsnap.deps: chartsnap
	@ if [ "$(CHART)" = "kong" ]; then \
		helm dependencies update charts/ingress; \
	fi
```